restapi/src/main/java/io/stargate/web/docsapi/resources/CollectionsResource.java

```diff
@@ -151,7 +139,10 @@ public Response createCollection(
           } else {
             return Response.status(Response.Status.CONFLICT)
                 .entity(
-                    String.format("Create failed: collection %s already exists", info.getName()))
+                    new Error(
+                        String.format(
+                            "Create failed: collection %s already exists.", info.getName()),
+                        Response.Status.CONFLICT.getStatusCode()))
```

Comment:
Would it make sense to create a utility class to avoid accidentally using different code in the `Response` (line 140) and `Error` (line 145) objects?

Reply:
It would, but I think this can be done in separate ticket/PR if we want, would be strange that I add it now and use it in only this place. In addition I see that in lot of places error is created without the status code, so this could be fixed as well.

Reply:
sgtm :+1:

Reply:
We can add that to https://github.com/stargate/stargate/issues/439

Reply:
OK will resolve this, and a note to #439.